### PR TITLE
docs: move Disable Resources docs to its own page

### PR DESCRIPTION
**Pull request by lizzthabet**
Move the beta Disable Resource documentation to its own page, so it's more visible and discoverable. I'm putting this up now so we can have a stable link to share with users. 

Besides the new page, the only other changes are:
* Added an explanation of "beta" at the top of the page (@lianmakesthings, lmk if this doesn't sound good!)
* The two minor notes about Disable Resources functionality are now formatted as bullet points, instead of tips, because they look too visually distracting in addition to the "beta" tip.

I'm not sure that it makes sense for these docs to be listed under the `Tiltfile` category, but we can move them later.

I need to do a little bit more work to support adding the "beta" tag to the nav and header, so I'll put that up in a separate PR.

### CODE REVIEW

**Comment by hyu** [comment]
@lizzthabet, this looks great! I think you can go ahead and add the Beta badge to the sidebar as well:
image

I did a quick edit in Inspector and it looks fine. Surbhi was on Zoom with me and agreed that it's good to ship for now, though we can refine it later by adjusting the colors. (i.e., when we decide to tag more stuff beta or experimental or such in our general docs, and thus want to color code)

```
<a class="Docs-subnav-link is-active" ...>
  Disabling Resources
  <span class="pill-tag">Beta</span>
</a>
```

**Review by hyu — APPROVED** [review]

File: docs/disable_resources.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+Depending on your `Tiltfile` setup, it may be important to note:
+* Disabling and enabling a resource doesn’t take into account any of its resource dependencies. For example, if resource B depends on resource A, disabling resource A will not disable resource B.
+* If you’re using [`tilt args`](tiltfile_config.html) to define a subset of resources Tilt is running, only the resources that are defined through args will show up in the UI Dashboard. In a future release, we’re planning to display all defined resources from the Tiltfile in the UI.
+
+## How to enable the feature
+While in beta, Disabling Resources feature requires opting in. First, make sure you're on [Tilt v0.23.5+](https://github.com/tilt-dev/tilt/releases). Then, add the following line to your Tiltfile:
+
+```
+enable_feature(“disable_resources”)
```
Comment (rwoll) [inline docs/disable_resources.md:22]:
```suggestion
enable_feature("disable_resources")
```

My editor was yelling at me when I copied the example in the current docs.

File: docs/disable_resources.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        }
+    }
+}
+```
+The configmap name will be under the UIResource’s `status.disableStatus.sources[0].configMap.name`.
+
+Now that you have the ConfigMap name, edit it with the desired disabled state:
+
+```shell
+$ tilt patch configmap glitch-disable -p ‘{ “data”: { “isDisabled”: “true” } }’
```
Comment (rwoll) [inline docs/disable_resources.md:102]:
```suggestion
$ tilt patch configmap glitch-disable -p '{ "data": { "isDisabled": "true" } }'
```

**Review by rwoll — CHANGES_REQUESTED** [review]
Carrying over the quoting changes as suggestions into this PR from https://github.com/tilt-dev/tilt.build/pull/1034.

File: docs/disable_resources.md (hunk 1 above)
Comment (lizzthabet) [reply docs/disable_resources.md]: ah, great catch! thanks. 🌟

**Comment by lizzthabet** [comment]
thanks, @hyu! the sidebar links / headers are also auto-generated, so i need to make template changes to get the pill-tags into those sections. i have this working locally and will put it up for pr separately.

File: docs/disable_resources.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+So, we’re planning these future features to make it easier to manage sets of resources, and pick up from where you left off every time you restart Tilt:
+- Disabling and enabling multiple resources at once from Table View
+- `tilt args` persistence, so that `tilt up` starts by default with the last used set of resources, similar to how your editor might reopen with the last set of open documents
+- Changing `tilt args` and `config.set_enabled_resources()` to use this new feature, rather than dropping non-enabled resources entirely; the Tilt UI will show all resources with only the specified resources enabled.
+- Defaulting resources to disabled rather than enabled, to give the user better control over which resources are running
+
+But! We might be wrong. Maybe you have another use case that we haven't considered? Please talk to us.
+* Slack us from the [Kubernetes #tilt channel](http://slack.k8s.io)
+* Send an email at [hi@tilt.dev](mailto:hi@tilt.dev)
```
Comment (lianmakesthings) [inline docs/disable_resources.md:116]:
```suggestion
* Send an email to [hi@tilt.dev](mailto:hi@tilt.dev)
```

File: docs/disable_resources.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+Now that you have the ConfigMap name, edit it with the desired disabled state:
+
+```shell
+$ tilt patch configmap glitch-disable -p '{ "data": { "isDisabled": "true" } }'
+```
+
+## What can I use this feature for? 
+We saw that for many cases, it’s not practical or feasible to run all your resources all the time. But changing resources can interrupt your flow (e.g., you have to switch between the UI and command line). Improving Tilt's interface can make it more seamless to change the resources you’re running at any given time, and reduce the cognitive load of finding and running the right resources you need to run.
+
+So, we’re planning these future features to make it easier to manage sets of resources, and pick up from where you left off every time you restart Tilt:
```
Comment (lianmakesthings) [inline docs/disable_resources.md:108]: This part doesn't seem to really match the title "What can I use this feature for". It seems like this section is really two different sections: "Why do I want to use this" and "How will I be able to use this in the future". Maybe add the latter as a smaller, sub title?

**Review by lianmakesthings — APPROVED** [review]

**Pull request merged**